adam-core/src/main/scala/org/bdgenomics/adam/models/SequenceDictionary.scala

```diff
@@ -191,7 +214,7 @@ object SequenceRecord {
   val REFSEQ_TAG = "REFSEQ"
   val GENBANK_TAG = "GENBANK"
 
-  implicit def ordering = SequenceOrdering
+  //implicit def ordering = SequenceOrderingByName
```

Comment:
this seems like an "oops" that snuck in via https://github.com/fnothaft/adam/pull/7. Can remove later